test/samples/dynamic-require-json/main.js

```diff
@@ -0,0 +1,6 @@
+function takeModule (withName) {
+	return require('./' + withName);
+}
+
+assert.deepEqual(takeModule('dyamic.json'), {value: 'present'});
```

Comment:
As we are checking specifically for the `.json` extension, I expected this to work once I included "rollup-plugin-json". Unfortunately, the helper used the `.json` extension for the wrapper module which results in a parse error. I hope we can make this work somehow e.g. by making sure the wrapper module is treated as a virtual module via the `\0` convention.